docs/source/using/profiles.adoc

```diff
@@ -31,10 +31,10 @@ There are two ways to create a new profile.
 
 [NOTE]
 ====
-Profile name can only consist of alphanumeric characters and *-* as a separator, for example:
+Profile name can only consist of alphanumeric characters (underscore *_* allowed) and *-* as a separator, for example:
 
-- foo, bar123, demo-profile and 1stProfile are considered as valid profile names.
-- $foo, bar#@, profile@123 are invalid profile names.
+- foo, bar_123, demo-profile and 1stProfile are considered as valid profile names.
```

Comment:
I find the used examples a bit too casual. I don't think it is good style to use things like foo and bar in the public facing documentation. Let's try to use some more real examples.